controllers/front/DiscountController.php

```diff
@@ -39,15 +39,26 @@ public function initContent()
     {
         parent::initContent();
 
-        $cart_rules = CartRule::getCustomerCartRules($this->context->language->id, $this->context->customer->id, true, false);
+        $cart_rules = CartRule::getCustomerCartRules($this->context->language->id, $this->context->customer->id, true, true, true);
```

Comment:
Is there any insight about why this takes place now? How should secret vouchers be distributed now? Had to roll this back in my case.